public/index.html

```diff
@@ -14,6 +14,11 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
```

Comment:
Can we change the title here?

Reply:
Good point! May be change to just `Deepseed` or do you have any touchy line @yelinaung ?

Reply:
So Deep So Seed ©

Reply:
and the favicon